src/transformers/models/t5/modeling_t5.py

```diff
@@ -1329,3 +1329,93 @@ def _reorder_cache(self, past, beam_idx):
 
             reordered_decoder_past = reordered_decoder_past + (reordered_layer_past_states,)
         return reordered_decoder_past
+
+    
+@add_start_docstrings(
+    "The bare T5 Model transformer outputting encoder's raw hidden-states" "without any specific head on top.",
+    T5_START_DOCSTRING,
+)
+class T5ModelEncoder(T5PreTrainedModel):
+    authorized_missing_keys = [
+        r"encoder\.embed_tokens\.weight",
+        r"decoder\.embed_tokens\.weight",
```

Comment:
no need for `decoder`

Reply:
done for all decoder keys.
Let me know if you have a better regex.